_scripts/instruction-widget/templates/install/snap.html

```diff
@@ -0,0 +1,36 @@
+<!-- The advanced instructions don't use snapd, so this warning isn't needed there -->
+{{^advanced}}
+<aside class="note">
+  <div class="note-header">
+    <h3>Snap Support</h3>
+  </div>
+  <p>
+  Certbot snap support is currently in its beta phase and only supports the
+  x86_64 architecture. You can find instructions for installing Certbot without
+  using snap by selecting your OS in the dropdown above.
+  </p>
+</aside>
+{{/advanced}}
+{{> header}}
+<li>
+    Install snapd
+    <p>
+    You'll need to install snapd and make sure you follow any instructions to enable classic snap support.<br>
+    Follow these instructions on <a href='https://snapcraft.io/docs/installing-snapd/'>snapcraft's site to install snapd</a>.
+    </p>
+    <p class="centered">
+    <a class="link-button" href='https://snapcraft.io/docs/installing-snapd/'>install snapd</a>
+    </p>
+</li>
+<li>
+    Remove any Certbot OS packages
```

Comment:
Can we give examples of how to uninstall certbot for some common cases of this?

Reply:
Good idea!

Do you have any strong opinions on styling/structure here? Ideas I had are:

* Just list a couple examples inline in the paragraph
* Do an unordered list like we do for `/etc/crontab/`, `/etc/cron.*/*`, etc. at https://certbot.eff.org/lets-encrypt/ubuntubionic-other
* Maybe try to make multiple blocks starting with the command prompt`$`? I putting them all in one block is a bad idea.

Reply:
either of the first two sounds good to me! agree not to do the last, unless we really want to try to carefully craft a single one that works across machines, but that seems not worth it. I think people tend to know which installation system they have available.

Reply:
I played with it a bit and like the last one the best. It looks like 
![Screen Shot 2020-04-21 at 4 24 11 PM](https://user-images.githubusercontent.com/6504915/79924418-98f5fd80-83ec-11ea-8684-df444cdd90c0.png)